src/lib/simulation.js

```diff
@@ -81,7 +81,7 @@ function incrementBy(_timeStep = params.timeStep) {
 function addActors(newActors) {
     Object.keys(newActors).forEach(actorId => {
         const actor = newActors[actorId];
-        const count = actor.count ? actor.count : 1;
+        const count = actor.count;
```

Comment:
same here as in `addActorsFromGui`: 
`const count = actor.count ? actor.count : 0;`